hooks/tk-multi-workfiles2/scene_operation_tk-nuke.py

```diff
@@ -331,10 +332,21 @@ def _scene_operation_hiero_nukestudio(
         """
         import hiero
 
+        engine = sgtk.platform.current_engine()
+        engine_context = engine.context
+        context_str = engine_context.serialize(engine_context)
```

Comment:
You can simply use `context_str = engine.context.serialize()`

Reply:
Addressed comment.